### PR TITLE
Replace miarroba.com with ads.miarroba.com

### PR DESCRIPTION
**Pull request by socram8888**
miarroba.com is the main page of the hosting company, which left it inaccessible while not blocking the ads at ads.miarroba.com.

### CODE REVIEW

**Comment by spirillen** [comment]
Well they are listed as `Malvertising filter list by Disconnect ` on disconnect

```
grep -in 'miarroba.com' simple_malvertising.txt
1874:miarroba.com
```

image

On the other hand a reason for not blocking it at [STLD](https://www.mypdns.org/w/domain/#stld) level is that is seems like Disconnect.me are the only one blocking them at STLD level

```
hosts-sources/data/yoyo.org/domain.list:ads.miarroba.com
hosts-sources/data/yoyo.org/domain.list:contadores.miarroba.com
hosts-sources/data/yoyo.org/domain.list:hitcounters.miarroba.com
hosts-sources/data/yoyo.org/domain.list:online.miarroba.com
hosts-sources/data/disconnect-me/domain.list:miarroba.com
hosts-sources/data/adaway/domain.list:ads.miarroba.com
hosts-sources/data/mvps/domain.list:ads.miarroba.com
```

But there is a hole different issue as well, if you try to visit a third level domain, like `http://amelie-sodomie.miarroba.com/` (Warning apparently **NSFW** domain) you'll be redirected to ` https://amelie-sodomie.miarroba.com/` with a SSL warning

image

Long story short: I believe this requires a bit more in depth investigation for a good decision.

My ref: <https://www.mypdns.org/T603>

**Comment by socram8888** [comment]
That "amelie-sodomie" website returns a SSL error because the website seems to no longer exist. If you ignore the SSL warning and proceed you'd see a 404 page, before being redirected to the root miarroba.com website.
